src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -51,6 +56,13 @@ const FolderViewBody = ({
 
   return (
     <>
+      {__TARGET__ === 'mobile' && (
+        <div>
+          {currentFolderId === ROOT_DIR_ID && <MediaBackupProgression />}
```

Comment:
I think it would be better to add these components in DriveView rather than in the generic file with a condition :)

Reply:
I'm wondering if we should not display this message on each views since the backup can have a big impact on the app performance... 🤔 

But OK, let's move this one on the driveview for now